subprojects/groovy-console/src/main/groovy/groovy/console/ui/Console.groovy

```diff
@@ -1136,13 +1142,25 @@ class Console implements CaretListener, HyperlinkListener, ComponentListener, Fo
         return consoleControllers.find { it.consoleId == consoleId }
     }
 
+    private enum SourceType {
+        GROOVY, JAVA
+    }
+
+    void runJava(EventObject evt = null) {
+        doRun(evt, SourceType.JAVA)
+    }
+
     // actually run the script
     void runScript(EventObject evt = null) {
```

Comment:
Would it be better to use default params here?
`void runScript(EventObject evt = null, SourceType st = SourceType.GROOVY) {`

Reply:
I am not sure whether swingbuilder knows such method signature or not...

Reply:
Groovy will create all the in between signatures, so it should work - we can try later.